app/src/main/java/org/systers/mentorship/view/adapters/MainPagerAdapter.kt

```diff
@@ -0,0 +1,23 @@
+package org.systers.mentorship.view.adapters
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import org.systers.mentorship.MentorshipApplication
+import org.systers.mentorship.R
+import org.systers.mentorship.view.fragments.*
+
+class MainPagerAdapter(fm: FragmentManager): FragmentPagerAdapter(fm) {
+
+    private val context = MentorshipApplication.getContext()
+
+    private val fragments = listOf(HomeFragment.newInstance(), ProfileFragment.newInstance(), RelationPagerFragment.newInstance(), MembersFragment.newInstance(), RequestsFragment.newInstance())
+    private val fragmentTitle = listOf(context.getString(R.string.fragment_title_home), context.getString(R.string.fragment_title_profile), context.getString(R.string.fragment_title_relation), context.getString(R.string.fragment_title_members), context.getString(R.string.fragment_title_requests))
```

Comment:
@Hardik0403 please refrain from using more than 100 characters in a line, this affects the readability of code